### PR TITLE
SPCTR-3468: Fix - Container background overlay style setting glitches.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Issue 1: Changing the Overlay Opacity slider doesn't do anything to the live page.Any changes I make to the Overlay Opacity slider is not reflected in the published page. See Video 1 for an example.I

Issue 2: When using the color picker for Image Overlay Color, it closes INSTANTLY after you pick a color.This makes it almost impossible to be precise when using the color picker. It also instantly closes if I manually enter a color's hex code. See Video 2 for an example.

Issue 3: I cannot click & drag the Overlay Opacity http://slider.at/  most, it will move a single notch. See Video 2 at 00:45 for an example.

Issue 4: Manually clicking & editing the text box of Overlay Opacity can crash the page.

Issue 5: Overlay color picker is not letting users to choose the color. The moment you click on it, it disappears.

Deleting all digits will crash the page. Highlighting the digits & entering a period ( . ) will crash the page. See Video 2 at 01:10 for an example.

### Screenshots

https://onedrive.live.com/?id=3BC1332C5FE0944B!3562&resid=3BC1332C5FE0944B!3562&authkey=!ABbuVXDtK_mTWyQ&cid=3bc1332c5fe0944b
https://onedrive.live.com/?authkey=%21AN4eOvxshh7hW7I&id=3BC1332C5FE0944B%213563&cid=3BC1332C5FE0944B&parId=root&parQt=sharedby&o=OneUp
### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/components/background/index.js [hunk 1]
```diff
@@ -138,7 +138,7 @@ const Background = ( props ) => {
 	};
 
 	const onSelectBgFallbackImage = ( media ) => {
-		if ( ! media || !media.url ) {
+		if ( ! media || ! media.url ) {
```
Comment (rahulvarma722) [inline src/components/background/index.js:141]:
```suggestion
		if ( ! media?.url ) {
```

File: src/components/background/index.js [hunk 2]
```diff
@@ -1524,21 +1515,24 @@ const Background = ( props ) => {
...
-					allow={ [ 'image' ] }
-					onSelectImage={ onSelectBgFallbackImage }
-					backgroundImage={ backgroundVideoFallbackImage.value }
-					onRemoveImage={ onRemoveBgFallbackImage }
-					disableDynamicContent={ true }
-					label={ __( 'Fallback Image', 'ultimate-addons-for-gutenberg' ) }
-					help={ __( 'This cover image will replace the background video in case that the video could not be loaded.', 'ultimate-addons-for-gutenberg' ) }
-				/>
-			</div>
+			{ 'video' === backgroundType.value && backgroundVideoType.value && (
```
Comment (rahulvarma722) [inline src/components/background/index.js:1518]:
```suggestion
			{ backgroundVideoType?.value && 'video' === backgroundType.value && (
```

**Review by rahulvarma722 — COMMENTED** [review]
Please verify GBS functionality also with these changes.

File: src/components/background/index.js (hunk 1 above)
Comment (rajmohanreddy774) [reply src/components/background/index.js]: Updated and tested.

File: src/components/background/index.js (hunk 2 above)
Comment (rajmohanreddy774) [reply src/components/background/index.js]: Updated and tested.

**Review by rahulvarma722 — APPROVED** [review]
Make sure GBS work properly

**Review by adi3890 — APPROVED** [review]

**Pull request merged**